regression/cbmc-incr-smt2/bitvector-bitwise-operators/shift_right.desc

```diff
@@ -0,0 +1,13 @@
+CORE
+shift_right.c
+--incremental-smt2-solver 'z3 --smt2 -in' --slice-formula
+\[main\.assertion\.1\] line \d+ This should be satisfiable: FAILURE
+\[main\.assertion\.2\] line \d+ This should also be satisfiable: FAILURE
+^EXIT=10$
+^SIGNAL=0$
+--
+--
+The assertions in the C file for this test are designed to be satisfiable,
```

Comment:
If they are satisfied then why are the regexes checking for failure? Or do you mean than we expect a `sat` response back from the solver, thus leading to a verification failure?

Reply:
Sorry, you're right, the wording is not ideal.

When I wrote this I had in mind the terminology from logic "the expression is satisfiable", which means that there's at least one model within the structure that satisfies the expression.

I'll reword this.

Reply:
Your current working still doesn't make sense to me. Your regexes are now looking for SUCCESS. So surely the solver will return `unsat` and there will be no counter examples found?